_data/rules.yml

```diff
@@ -294,7 +294,16 @@ services:
 
   - name: Kubernetes
     exporters:
-      - rules:
+      - name: kubelet
+        rules:
+          - name: Volume out of disk space
+            description: Volume is almost full (< 10% left)
+            query: 'kubelet_volume_stats_available_bytes / kubelet_volume_stats_capacity_bytes * 100 < 10'
+            severity: warning
+          - name: Volume full in four days
+            description: "{{ $labels.namespace }}/{{ $labels.persistentvolumeclaim }} is expected to fill up within four days. Currently {{ $value | humanize }}% is available."
+            query: '100 * (kubelet_volume_stats_available_bytes / kubelet_volume_stats_capacity_bytes) < 15 and predict_linear(kubelet_volume_stats_available_bytes[6h], 4 * 24 * 3600) < 0'
+            severity: critical
```

Comment:
Should be `error` instead of critical.

In this file, we only have 2 severity levels ;)